testnet/stacks-node/src/neon_node.rs

```diff
@@ -1251,7 +1251,7 @@ impl BlockMinerThread {
         chain_state: &mut StacksChainState,
         at_stacks_height: Option<u64>,
     ) -> Option<TipCandidate> {
-        info!("Picking best Stacks tip");
+        debug!("Picking best Stacks tip");
```

Comment:
The reason these are all `info` is that this is pertinent information for miners who need to debug their chain-tip selection without turning on the debug firehose.

Reply:
The trouble is that this is printing every 100ms.

Reply:
So it becomes a firehose on its own.

Reply:
Why are you mining every 100ms?

Reply:
That's still buggy behavior (or buggy configuration).  The node shouldn't be trying to build blocks so quickly.

Reply:
Yeah, you're right. Will try to debug that issue separately. Glad to hear that this is not expected to constantly do this! I'll close this PR.